design-at-cornell/src/events/EventsStyles.ts

```diff
@@ -119,41 +126,20 @@ export const Settings = styled.div`
     line-height: 23px;
     color: #939393;
     margin: 0 30px 0 0;
+    overflow: visible;
+    white-space: nowrap;
+    display: inline-block;
+    text-overflow: clip;
+    width: calc(200%);
   }
 `;
 
-export const TagsContainer = styled.div`
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  width: 100%;
-  height: fit-content;
-  align-items: flex-start;
-`;
-
-export const Tag = styled.div`
-  display: flex;
-  width: fit-content;
-  height: 22px;
-  border-radius: 43px;
-  margin-right: 10px;
-  padding: 10px;
-  align-items: center;
-  background: #e4e4e4;
-  p {
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 14px;
-    color: rgba(0, 0, 0, 0.49);
-  }
-`;
-
-export const Link = styled.div<{ shadowColor: string }>`
+export const EventLink = styled.div`
   display: flex;
   width: fit-content;
   height: 49px;
   border-radius: 9px;
-  box-shadow: 0px 0px 22px ${({ shadowColor }) => shadowColor};
+  box-shadow: 0px 0px 22px #aed8f7;
```

Comment:
Instead of writing the color hex code here, you can import the color hex from `colors.ts`:

Have `import { colors } from '../constants/colors';` at the top of this file, and use `${colors.event_link_blue}` instead of `#aed8f7`. This makes the code more clean.